mlflow/cli.py

```diff
@@ -486,6 +486,7 @@ def gc(backend_store_uri, run_ids):
         artifact_repo = get_artifact_repository(run.info.artifact_uri)
         artifact_repo.delete_artifacts()
         backend_store._hard_delete_run(run_id)
+        # pylint: disable=print-function
```

Comment:
can we try the click echo here to see if it functions how we want it to?

``` python
click.echo(f"Run with ID {str(run_id)} has been permanently deleted.")
```

Reply:
Updated the code to use `click.echo`!